webapp/src/components/tutorial/TutorialContainer.tsx

```diff
@@ -106,9 +109,77 @@ export function TutorialContainer(props: TutorialContainerProps) {
     const markdown = steps[visibleStep].headerContentMd;
     const hintMarkdown = steps[visibleStep].hintContentMd;
 
+    const validateTutorialStep = async () => {
+        const globalValidators = tutorialOptions.globalValidationConfig?.validators
+                                    ?? PopulateValidatorCache(tutorialOptions.globalValidationConfig);
+        const localValidators = currentStepInfo.localValidationConfig?.validators
+                                    ?? PopulateValidatorCache(currentStepInfo.localValidationConfig);
+
+        let validators: pxt.Map<CodeValidator> = {}
+        if(localValidators) {
+            Object.entries(localValidators).forEach(v => validators[v[0]] = v[1]);
+        }
+        if (globalValidators) {
+          Object.entries(globalValidators).forEach(v => {
```

Comment:
nit: looks like there's a bit of a mix of 4 space tabs and 2 space tabs in this file, should be 4 across the board~

Reply:
lol good catch. I had prettier set to 2 spaces and VS Code's default formatter set to 4 🙄